utils/models.py

```
@@ -0,0 +1,178 @@
+
+    def get_output(self, tokens):
+        with torch.no_grad():
+            return self.model(**tokens)
+
+
+class PoliticalBiasBert(Model):
+    def __init__(self):
+        super().__init__(
+            AutoTokenizer.from_pretrained("bert-base-cased"),
+            AutoModelForSequenceClassification.from_pretrained(
+                "bucketresearch/politicalBiasBERT"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)